includes/blocks/course-theme/class-course-content.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * File containing the Sensei\Blocks\Course_Theme\Course_Content class.
+ *
+ * @package sensei
+ * @since 4.0.0
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Block to render the content for the current lesson or quiz page.
+ */
+class Course_Content {
+
+	/**
+	 * Content constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/course-content',
+			[
+				'render_callback' => [ $this, 'render_content' ],
+			]
+		);
+	}
+
+	/**
+	 * Render content for the current page.
+	 *
+	 * @access private
+	 *
+	 * @return string HTML
+	 */
+	public function render_content() {
+		$type = get_post_type();
+
+		switch ( $type ) {
+			case 'quiz':
+				return Quiz_Content::render_quiz();
```

Comment:
You forgot to import the `Quiz_Content` class. Like `use \Sensei\Blocks\Course_Theme\Quiz_Content`.

It doesn't break anything the way it is. But, just in case in the future we tweak our autoloader it's better if namespaces are used consistently.

Reply:
Would that be needed for some autoloader setup? They are in the same namespace. I'm just thinking about replacing all the individual `use` statements elsewhere to reduce some noise, like below. Do you think that could be breaking things going forward?

```php
use \Sensei\Blocks\Course_Theme as Blocks;
...
new Blocks\Course_Title();
new Blocks\Site_Logo();
```

Reply:
Actually went ahead and did this ^ in 910f2c1, since this PR is already changing that list.

Reply:
Actually, I agree. I think it's better this way. Thanks for doing this! 💪